pandas/tests/frame/methods/test_rename.py

```diff
@@ -164,7 +164,7 @@ def test_rename_multiindex(self):
 
     def test_rename_nocopy(self, float_frame):
```

Comment:
can you parameterize this both with and w/o copy

Reply:
the test below this has the copy=True test, but then also has some other stuff.  will look at splitting that test and then parametrizing in a separate pass